lib/rack/attack/store_proxy/redis_cache_store_proxy.rb

```diff
@@ -15,7 +15,7 @@ def increment(name, amount = 1, options = {})
           #
           # So in order to workaround this we use RedisCacheStore#write (which sets expiration) to initialize
           # the counter. After that we continue using the original RedisCacheStore#increment.
-          if options[:expires_in] && !read(name)
+          if options[:expires_in] && !read(name, raw: true)
```

Comment:
This of course fixes this particular `read` invocation, but in order to fix them all I think we need to override `#read` with `raw: true`, similar to what we have for `#write`.

Reply:
...and that would mean this particular change can be reverted.

Reply:
Ah, thank you! Given that `write` has been overridden so it always uses `raw: true`, makes sense that `read` would need to do the same – and I can see the project is already doing this for `ActiveSupportRedisStoreProxy` 🙂 PR update coming shortly.